Sources/CoreData/CoreDataSource.swift

```diff
@@ -302,7 +302,8 @@ public class CoreDataSource<T: NSManagedObject & DataObject>: NSObject, NSFetche
     }
 
     private func sectionInfo(for index: Int) -> NSFetchedResultsSectionInfo? {
-        return controller.sections?[safe: index]
+        guard let sections = controller.sections, index < sections.count else { return nil }
```

Comment:
Why are you moving away from `safe` in these two places?

Reply:
The `Collection+Extensions` file is in Flapjack itself, and I didn't want to make _it_ `public` because it shouldn't be exposing that kind of detail, and I didn't want to spin out a separate `FlapjackSupport` module (that Flapjack and FlapjackCoreData could include, but not expose when other code imports _it_) pretty much just for this one extension that got shared.

Does that make sense? I feel like I just typed in circles.

Reply:
I'm not sure why you would need to expose that extension just to use it internally, but it's not a big deal either way...

Reply:
I guess because you're exporting Flapjack as code, not as a pre-built framework...

Reply:
Well, and there's a bunch of good code that `import Flapjack` brings into your project — and we wouldn't want to have that also import what amounts to an internal implementation detail, because that implementation detail (the extension) is _also_ in the `Flapjack` framework.